### ISSUE TITLE
Accessibility: Code blocks are not announced/landmark

### ISSUE THREAD

**Issue by ghost** [opentofu/registry-ui#155]
The code blocks are not currently announced. Please take a look at opentofu/opentofu.org#260 and the OpenTofu website and format the code blocks accordingly.
